tests/integration/util_test.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 var (
+	waitForIpTime    = 120 * time.Second
```

Comment:
Your commit says "1m to be on safe side" but this is two minutes, unless I'm really tired. :-)
I don't have a strong preference but I'd like to see the commit and code agree.

Reply:
Fixed and updated in #162